packages/react/src/components/form/initial/oidc.tsx

```diff
@@ -15,9 +15,9 @@ import { Line } from "../../icon/line";
 
 import * as styles from "./initial.css";
 
-const PROVIDER_TO_ICON: Record<string, React.ReactNode> = {
+const PROVIDER_TO_ICON: Record<OAuthProvider, React.ReactNode> = {
```

Comment:
This type change means we get build errors if a new provider is added which we don't have a pretty name or icon for.